components/helpers/parent-by-attribute.js

```diff
@@ -0,0 +1,16 @@
+define('parent-by-attribute', [], () =>
+    function getParentByAttribute(el, attName, attValue) {
```

Comment:
JSDoc? Blimey this is a monstrosity though. 

Add a TODO: replace with [Element.closest()](http://caniuse.com/#search=closest) and its [polyfill](https://github.com/jonathantneal/closest) for IE.